src/inc/palclr.h

```diff
@@ -484,7 +484,14 @@
 //  and it provides __declspec(selectany) to instruct the linker to merge
 //  duplicate external const static data copies into one.
 //  
+#if defined(__clang__)
+#define SELECTANY extern
```

Comment:
@adiaaida The proposal is to add !PLATFORM_UNIX to the condition, not to define it during the translation.  At any rate, as we discussed offline, perhaps it makes sense to define a specific macro *only* during formatting to make the intent clear.